cdn/public/cli/install.sh

```diff
@@ -1,59 +1,93 @@
 #!/usr/bin/env bash
```

Comment:
Since the install script is BASH, should we use `bash` instead of `sh` for the install command?
```
curl -sSL https://install.apollographql.workers.dev/cli | bash
```